test/unicode/utf32.jl

```diff
@@ -62,13 +62,12 @@ tstcvt(str4_UTF8,str4_UTF16,str4_UTF32)
 # Test converting surrogate pairs
 @test utf16(strS_UTF8) == strC_UTF8
 @test utf32(strS_UTF8) == strC_UTF8
-@test utf8(strS_UTF16) == strC_UTF8
+@test String(strS_UTF16) == strC_UTF8
 @test utf32(strS_UTF16) == strC_UTF8
-@test utf8(strS_UTF32)  == strC_UTF8
+@test String(strS_UTF32)  == strC_UTF8
 @test utf16(strS_UTF32) == strC_UTF8
 
 # Test converting overlong \0
-@test utf8(strZ)  == strz_UTF8
```

Comment:
this should probably still test that `convert(String, strZ) == strz_UTF8` for a faithful deprecation